clients/unshaded/src/main/java/tachyon/client/ReadType.java

```diff
@@ -16,28 +16,28 @@
 package tachyon.client;
 
 /**
- * Different read types for a TachyonFile.
+ * Convenience modes for commonly used read types for a TachyonFile.
  *
- * As of 0.8, replaced by {@link TachyonStorageType} and {@link UnderStorageType}
+ * For finer grained control over data storage, advanced users may specify
+ * {@link tachyon.client.TachyonStorageType} and {@link tachyon.client.UnderStorageType}.
  */
-@Deprecated
 public enum ReadType {
   /**
-   * Read the file and but do not cache it explicitly.
+   * Read the file and skip Tachyon storage. This mode will not alter any data distribution
+   * in Tachyon storage.
    */
   NO_CACHE(1),
   /**
-   * Read the file and cache it.
+   * Read the file and cache it in a local worker. This read type will not move data between
+   * tiers of Tachyon Storage. Users should use CACHE_PROMOTE for more optimized performance with
+   * tiered storage.
```

Comment:
Does this mean it's not guaranteed where tier this data will land? Shall we provide some more description on its behavior?